pyproject.toml

```diff
@@ -197,6 +214,22 @@ keep-runtime-typing = true
     "RUF012",  # Mutable class attributes should be annotated with `typing.ClassVar`
     "UP031",  # Use format specifiers instead of percent format
 ]
+"ops/_tracing/vendor/*" = [
```

Comment:
If this is a pure vendor (the code being exactly the same as upstream) then probably simpler to just turn checking entirely off for the whole vendor folder, like with format?